mvp-spec/operator-api.md

```diff
@@ -559,48 +752,62 @@ This endpoint doesn't rely on support of 3PC or not: the REST version will work
 - **no** signature verification
 - if `paf_test_3pc` exists, return `true`. Otherwise, return `false`
 
-On a call to `GET /v1/id-prefs`, when no cookie is found on PAF TLD+1 domain,
+On a call to `GET /v1/ids-prefs`, when no cookie is found on PAF TLD+1 domain,
 the operator attempts to write a short-life `paf_test_3pc` cookie.
 
-This endpoint is **only** called immediately after a call to `GET /v1/id-prefs` has failed, to
+This endpoint is **only** called immediately after a call to `GET /v1/ids-prefs` has failed, to
```

Comment:
FAR: not convinced this optimization is reasonable (vs having an autonomous endpoint that redirect to itself to verify 3PC functionality)

Reply:
Let's discuss this in a dedicated issue, since I don't know what cons you see in this approach.